gateway-js/package.json

```diff
@@ -29,15 +29,14 @@
     "@apollo/core-schema": "~0.3.0",
     "@apollo/federation-internals": "file:../internals-js",
     "@apollo/query-planner": "file:../query-planner-js",
+    "@apollo/server-gateway-interface": "https://pkg.csb.dev/apollographql/apollo-server/commit/bdd19924/@apollo/server-gateway-interface",
+    "@apollo/usage-reporting-protobuf": "https://pkg.csb.dev/apollographql/apollo-server/commit/bdd19924/@apollo/usage-reporting-protobuf",
```

Comment:
(This might get changed back to `apollo-reporting-protobuf` until we have a non-alpha of the above, although the package is pretty much unchanged other than name and it doesn't have any dependencies.)